src/style/ImpactCalculator.less

```diff
@@ -184,3 +190,7 @@
 .facebook-btn:hover {
 	color: @color-facebook;
 }
+
+.header .description-short p {
+	font-size: 1.143em;
```

Comment:
We generally want to avoid local font settings -- it makes for inconsistent typography. So when you need to set one, please add a comment to say why this `p` differs from normal `p`s. Thanks.